content/tutorials/hello-world-node.md

```diff
@@ -35,320 +36,459 @@ Node.js installation varies depending on your operating system.
 | Linux                  | Use the instructions provided by your **Linux** package manager. <br/> Node.js maintains a list of <a href="https://nodejs.org/en/download/package-manager" target="_blank">supported packages</a>.  |
 
 
-#### Install Rust
 
-Rust language utilizes an installer to download and provision Rust on your local system. Refer to <a href="https://rustup.rs" target="_blank">rustup</a> documentation to instructions.
+## Application Development
 
-##### Install Rust toolchain
+This section provides a step-by-step on how to build a simple data streaming app using Node.js. If you'd like to download the demo app instead, skip ahead to [Download Fluvio Demo Apps](#download-node-demo-apps).
+
+
+### Initialize a New Node.js Project
 
-Fluvio compiler uses the nightly toolchain. To install, run:
+Create a directory for **fluvio-node-app**:
 
 ```bash
-$ rustup toolchain install nightly
- ...
-nightly-x86_64-apple-darwin installed - rustc 1.44.0-nightly (f509b26a7 2020-03-18)
+$ mkdir fluvio-node-app
+$ cd fluvio-node-app
 ```
 
-Make nightly toolchain default:
+Run npm to create a **node project** and generate package.json file:
 
 ```bash
-$ rustup default nightly
-info: using existing install for 'nightly-x86_64-apple-darwin'
-info: default toolchain set to 'nightly-x86_64-apple-darwin'
-
-nightly-x86_64-apple-darwin unchanged - rustc 1.44.0-nightly (f509b26a7 2020-03-18)
+$ npm init -y
 ```
 
+### Updating the Project for TypeScript
 
-##### Install Node build tool
-
-The **nj-cli** is a tool that generates native Node interfaces for Rust APIs. Use  **cargo** to install nj-cli tool:
+The Fluvio Node.js client provides TypeScript definitions. Let's setup our demo application to use TypeScript.
 
 ```bash
-$ cargo install nj-cli
-Updating crates.io index
-...
-Installing /Users/user/.cargo/bin/nj-cli
-Installed package `nj-cli v0.1.2` (executable `nj-cli`)    
+# Install TypeScript locally to the project as a development dependency;
```

Comment:
I think comments look too dark in the bash code blocks. Can we move the comments to just plain text?

![image](https://user-images.githubusercontent.com/4210949/94999446-7bc27f00-0587-11eb-80d1-674f3d0f87da.png)

Reply:
commit 3a13337 now includes comments in plain text